src/main/java/file/methods/FileHandler.java

```diff
@@ -0,0 +1,83 @@
+package file.methods;
+
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+import tasks.Task;
+import tasks.Todo;
+import tasks.Event;
+import tasks.Deadline;
+
+public class FileHandler {
+
+    public static void setUpFile(ArrayList<Task> actions) {
+        try {
+            File f = new File("data/savedList.txt");
+            File directory = new File("data");
```

Comment:
Perhaps avoid the usage of magic literals. You can store the string literals as constants and use them (final static vars). [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)